src/OpenRiaServices.Hosting.AspNetCore/Framework/AspNetCore/Operations/OperationInvoker.cs

```diff
@@ -77,17 +78,23 @@ protected async Task<(ServiceQuery, object[])> ReadParametersFromBodyAsync(HttpC
                 throw new BadHttpRequestException("invalid lenght", (int)System.Net.HttpStatusCode.LengthRequired);
 
             var length = (int)contentLength;
-            var buffer = ArrayPool<byte>.Shared.Rent(length);
+
```

Comment:
note: it would be quite easy to remove requirement of having content-lenght specified after these changes